hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieDeltaStreamer.java

```diff
@@ -511,6 +524,22 @@ public void testNullSchemaProvider() throws Exception {
     }
   }
 
+  @Test
+  public void testJdbcbasedSchemaProvider() throws Exception {
```

Comment:
can we create a separate test class for this? given you are only testing the schema provider?